tools/generate_hotkeys.py

```diff
@@ -26,14 +35,34 @@
             "|Command|Key Combination|\n"
             "| :--- | :---: |\n"
         )
+        keys_list = []
+        for help_text, key_combinations_list in categories[action].items():
+            keys_list.append(key_combinations_list)
```

Comment:
Here, do you mean to `extend` instead of `append`? As appending would not check if, `q` for example is also in ['ctrl a', 'q']

Reply:
I will change it to extend

Reply:
Not that you have to change the rest of the `key_list` logic as well.